.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,30 @@
+name: Create Release PR
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Create Release PR
+    runs-on: ubuntu-latest
+    if: ${{ !startsWith(github.event.head_commit.message, 'Version Packages') }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js 18
```

Comment:
It says 18.x in the next workflow. Could you keep the names consistent? (either one is fine with me)

Reply:
Set to 18.